src/client/clientevent.h

```diff
@@ -153,6 +156,16 @@ struct ClientEvent
 			std::string *type;
 			std::vector<std::string> *params;
 			bool clouds;
+			video::SColor *day_sky;
```

Comment:
gross - pointers here are largely unnecessary. If you want to reduce the size of the uninon, use `std::unique_ptr`